From 79ee5833477a24fe431759b27644506098447352 Mon Sep 17 00:00:00 2001
From: tgeisler <tgeisler@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 28 Apr 2011 16:15:21 +0000
Subject: [PATCH 0032/1000] Mantis0018904: mcontrol hung if corba servant is
 created too early Mantis0018811: mcontrol should wait a given time till
 startup Mantis0013984: Update-Service should show a message when update is
 carried out Mantis0014040: Update-Service should show a message when update
 is carried out

git-svn-id: svn://localhost/SelfServiceCommon/trunk@137 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/cpp/Controller/inc/mcontrol_def.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/SelfServiceCommon/Massai/cpp/Controller/inc/mcontrol_def.h b/SelfServiceCommon/Massai/cpp/Controller/inc/mcontrol_def.h
index a2b22d2b..93847d9a 100644
--- a/SelfServiceCommon/Massai/cpp/Controller/inc/mcontrol_def.h
+++ b/SelfServiceCommon/Massai/cpp/Controller/inc/mcontrol_def.h
@@ -25,8 +25,8 @@
  *                   \li      - mcontrol_small_ds.bmp is shown, has to be placed in the working folder of mcontrol.exe
  *                   \li QA-Tests: Check window shown while start is delayed
  *                   \li QA-Tests: Check window shown while update is carried out
- *                   \li QA-Tests: Check all windows shown in server and kiosk mode
- *                   \li QA-Tests: Check if platform startup with high kiosk load
+ *                   \li QA-Tests: Check all windows shown and start funtions in server and kiosk mode
+ *                   \li QA-Tests: Check platform startup with high kiosk load
  *                   \li QA-Tests: Check if several times start/stop of platform is reliable
  *
  *
-- 
2.41.0.windows.1

